pkg/cmd/pr/shared/templates.go

```diff
@@ -97,6 +149,35 @@ func hasIssueTemplateSupport(httpClient *http.Client, hostname string) (bool, er
 	return hasQuerySupport && hasMutationSupport, nil
 }
 
+func hasPullRequestTemplateSupport(httpClient *http.Client, hostname string) (bool, error) {
```

Comment:
This function does the exact same query as `hasIssueTemplateSupport`, except that it scans for a different field on a Repository object. Could this lookup instead piggyback on `hasIssueTemplateSupport` so that fewer API calls are done?

Reply:
Yup good call. I consolidated `hasIssueTemplateSupport` and `hasPullRequestTemplateSupport` into a single `hasTemplateSupport` function which uses one query and supports both issues and pull requests.